pallets/claims/src/claims_data.rs

```diff
@@ -0,0 +1,10 @@
+use primitives::Balance;
+use sp_std::vec;
+lazy_static::lazy_static! {
+pub static ref CLAIMS_DATA: vec::Vec<(&'static str, Balance)> = vec![
+	("0x8202c0af5962b750123ce1a9b12e1c30a4973557", 555),
```

Comment:
We'll replace this with real data right?

Reply:
yes, if those can be published (otherwise we could restrict file permissions on github or other solution in mind?)

Reply:
Of course they need to be published :)